policy/centos9/scripts/sign

```diff
@@ -36,5 +38,10 @@ case "$RPM_CHANNEL" in
 esac
 
 set +x
+
 echo "Signing RPMs with ci@rancher.com's GPG KEY"
+
 rpmsign --addsign dist/centos9/**/rancher-*.rpm --define "_gpg_name ci@rancher.com" --define "_gpgbin /usr/bin/gpg" --define "__gpg_sign_cmd %{__gpg} gpg --force-v3-sigs --batch  --no-armor --pinentry-mode loopback --passphrase "$PRIVATE_KEY_PASS_PHRASE" -u "%{_gpg_name}" -sbo %{__signature_filename} --digest-algo sha256 %{__plaintext_filename}"
+
+# Check signature in rpms
+for rpm in $(find dist/centos9/ -type f -name "*.rpm"); do rpm --checksig --verbose $rpm || true; done
```

Comment:
The `|| true` seems to go in the counter direction of having the signature check - as folks may not pay attention to error messages if the process does not actually fail.

Is there any specific reason why we need to ignore failures here?

Reply:
@pjbgf good point, for now `true` is only there to avoid the `sign` script to fail, but still gets some information about one RPM's signature with `rpm --checksig`.  For example, rpm --checksig returns 1 if the public key is not installed on the OS (NO KEY), which is expected in the current setup.
I've updated the script ([commit](https://github.com/rancher/rancher-selinux/pull/24/commits/9917de7a8d79eb2dcf0cc52b23b67557afc1c04e)) with some comments to make it more clear. The comments also explain the possible errors that `rpmsign --checksig --verbose $rpm` could return. We can then improve the tests based on these output errors, later.
Note that RPM's signature through rpmsign's exec verification is still tested though.